lib/components/map/default-map.js

```diff
@@ -56,6 +108,13 @@ class DefaultMap extends Component {
     setLocation(payload)
   }
 
+  componentDidUpdate (prevProps) {
+    // TODO: reinstate this call below.
+    // this._updateBounds(prevProps, this.props)
```

Comment:
The bounds updating logic has been moved to the `BoundsUpdatingOverlay` component. I think this is already accounted for.

Reply:
Great. I have updated the comment.